gemini_report_writer/agents/quality_pipeline.py

```
@@ -0,0 +1,582 @@
+import json
```

Ruff (0.11.9)

`json` imported but unused

Remove unused import: `json`

(F401)